base/linalg/bidiag.jl

```diff
@@ -40,6 +40,15 @@ function getindex{T}(A::Bidiagonal{T}, i::Integer, j::Integer)
     i == j ? A.dv[i] : (A.isupper && (i == j-1)) || (!A.isupper && (i == j+1)) ? A.ev[min(i,j)] : zero(T)
 end
 
+## structured matrix methods ##
+function Base.replace_in_print_matrix(A::Bidiagonal,i::Integer,j::Integer,s::AbstractString)
+  if A.isupper
```

Comment:
ref https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#code-formatting-guidelines

What editor are you using? 4-space indent is a good default to set for contributing to Julia, and most packages follow the same convention.